src/karmen_frontend/cypress/integration/printers/listing_spec.js

```diff
@@ -28,24 +28,32 @@ describe("Printers: Listing", function () {
   });
 
   it("has the create button", function () {
-    cy.get(".react-tabs__tab-panel__header a").should(
-      "have.attr",
-      "href",
-      `/${organizationUuid}/add-printer`
-    );
+    cy.get("input#username").type(email);
+    cy.get("input#password").type(password);
+    cy.get('button[type="submit"]').click().wait(5000).then(() => {
+      cy.get(".react-tabs__tab-panel__header a").should(
```

Comment:
This is extremely prone to HTML changes. Again, we should select by semantic queries.

Reply:
After internal discussion, we've decided not to block approval on this one and instead forward this to the upcoming e2e test suite upgrade.